scenariorun/src/main/kotlin/com/cosmotech/scenariorun/containerregistry/DockerContainerRegistryClient.kt

```diff
@@ -0,0 +1,24 @@
+// Copyright (c) Cosmo Tech.
+// Licensed under the MIT license.
+package com.cosmotech.scenariorun.containerregistry
+
+import com.cosmotech.api.config.CsmPlatformProperties
+import com.cosmotech.api.containerregistry.RegistryClient
+import org.springframework.boot.autoconfigure.condition.ConditionalOnProperty
+import org.springframework.stereotype.Service
+
+@Service("csmDockerContainerRegistry")
+@Suppress("UnusedPrivateMember")
+@ConditionalOnProperty(
+    name = ["csm.platform.containerRegistry.provider"],
+    havingValue = "local",
+    matchIfMissing = true)
+class DockerContainerRegistryClient(private val csmPlatformProperties: CsmPlatformProperties) :
+    RegistryClient {
+
+  override fun getEndpoint() = "Not yet implemented"
+
+  override fun checkSolutionImage(repository: String, tag: String) {
+    // TODO("Not yet implemented")
+  }
```

Comment:
I'm not sure we still need this...
If not, you can remove it.
If it's needed (in a way that we want to check existing solution in a cluster local):
- we need an implementation... maybe have another look on `docker-java` or check if something similar exists
- this class should be in `common`
- an interface should be define and this class should implement it (same thing for `AzureContainerRegistryClient` in `cosmotech-api-azure`)
- interface should be used instead of the azure client